python/cudf/cudf/core/frame.py

```diff
@@ -1195,7 +1195,7 @@ def scatter_by_map(
             map_index = as_column(map_index)
 
         # Convert float to integer
-        if map_index.dtype == np.float:
+        if map_index.dtype == float:
```

Comment:
This feels funky. Should this be `np.float32` or something similar?

Reply:
This type-cast has to be made for all float types, and not just for one of `float64` or `float32`. Updated the check.